meta-intel-edison-bsp/recipes-kernel/linux/files/boottime_trace.cfg

```diff
@@ -1,4 +1,4 @@
 CONFIG_BOOT_CONFIG=y
 CONFIG_BOOTTIME_TRACING=y
 CONFIG_BOOT_CONFIG_EMBED=y
-CONFIG_BOOT_CONFIG_EMBED_FILE="/home/ferry/tmp/edison-intel/my/edison-morty/out/linux64/build/dwc3_tracer.txt"
+CONFIG_BOOT_CONFIG_EMBED_FILE="/home/ferry/tmp/edison/Documentation/dwc3/dwc3_tracer.txt"
```

Comment:
How others can you this recipe if it has the path being hard coded? I would suggest to change to ~ or $HOME at the bare minimum.

Reply:
Yes, this is ugly. This cfg is commented out in the kernel recipies, so this doesn't break anything. But in case in future a tracer needs to be built in the path needs to be manually fixed. I believe the kernel needs an absolute path here? But I'll follow your suggestion and in addition put a comment near the include.